cmake/swigResqml2JavaInclude.i

```diff
@@ -87,6 +87,18 @@ Basically this file add methods resqml2_0_instantiate* which will create the rig
     {
         ret = new com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.TimeSeries(cPtr, owner);
     }
+	else if (type.equals("GraphicalInformationSet"))
```

Comment:
it would be good to add those lines to CSharp swig file as well.

Reply:
Done